reactDashboard/src/pages/DataSources.jsx

```diff
@@ -99,7 +100,7 @@ const DataSources = () => {
           </a>
         </li>
       </ul>
-      <hr />
+      <hr className="full-width-hr"/>
```

Comment:
Same as with Accessibility but not as bad, the extra "full-width-hr" doesn't help my eyes flow down the page on dark mode. I'd suggest removing the one after the heading only if the Accessibility ones are removed.

Reply:
✅